Neos.ContentRepository.Core/Classes/Feature/RebaseableCommand.php

```diff
@@ -19,7 +21,7 @@
 final readonly class RebaseableCommand
 {
     public function __construct(
-        public RebasableToOtherWorkspaceInterface $originalCommand,
+        public (RebasableToOtherWorkspaceInterface&CommandInterface)|(RebasableToOtherWorkspaceInterface&SerializedCommandInterface) $originalCommand,
```

Comment:
This is some crazy type hinting here, but..

Reply:
I wonder if it's even necessary? Ultimately we want a  `RebasableToOtherWorkspaceInterface` here? do we even need the union?

Reply:
@kitsunet good point.. Like this: c36073ea9cd80dae992b0fdceb5d671795ad5406 ?

Reply:
Ah, dang, that breaks the UI
![image](https://github.com/user-attachments/assets/3738ccd5-2c0f-48b9-8931-1e20534c2581)